docs/source/monitors.rst

```diff
@@ -17,24 +17,140 @@ These monitors build off of a previous COS Branch monitoring program, an overvie
 
 Target Acquisition Monitors
 ---------------------------
-The goal of the Target Acquisition monitors is to assist in cases of failed acquisitions as well has keep track of slew
-vs time in order to identify any systematic trends or issues with COS target acquisitions.
-
-Each acquisition monitor is covers both slew directions (ACQSLEWX and ACQSLEWY) and according to which FGS was used.
+The goal of the Target Acquisition monitors is to assist in cases of failed acquisitions as well as keep track of
+performance order to identify any systematic trends or issues with COS target acquisitions.
 
 ACQIMAGE Monitor
 ^^^^^^^^^^^^^^^^
+.. note::
+
+    This monitor is still in development.
+    Output and tracking will be described in more detail once it is finalized.
+
+The ``AcqImage`` monitor is intended to be a more generalized monitor that keeps track of acquisition statuses and slew
+statistics.
 
-.. Needs to be finalized
+This monitor is run at a monthly cadence.
```

Comment:
I think this may be the only place where the cadence is mentioned. You could add it to some of the others, or maybe it would be better to just have a table somewhere listing a cadence of all the monitors. Then it would be easier to remember to change it if we change one of them.